pkg/loki/modules_test.go

```diff
@@ -216,7 +216,50 @@ func TestIndexGatewayRingMode_when_TargetIsRead(t *testing.T) {
 		})
 
 	})
+}
+
+func TestIndexGatewayClientConfig_when_TargetIsQuerier(t *testing.T) {
```

Comment:
We are testing both the read and the querier target here. So the test name should not be `...TargetIsQuerier`.
Maybe `TestIndexGatewayClientConfig_when_TargetIsQuerierOrRead`?